deepchem/models/openvino_model.py

```diff
@@ -0,0 +1,86 @@
+import os
+import subprocess
+import numpy as np
+import tensorflow as tf
+from tensorflow.python.framework.convert_to_constants import convert_variables_to_constants_v2
+
+try:
+  import mo_tf
+  from openvino.inference_engine import IECore
+  is_available = True
+except:
+  is_available = False
+  pass
+
+
+class OpenVINOModel:
+
```

Comment:
Could you add OpenVINO to the docs/models.rst file so we get generated autodoc?

Reply:
As for me, OpenVINOModel seems to be extra in the list. It's just a backend which can be enabled for other Keras (Done) or PyTorch models (TBD).

Reply:
Added Torch models inference. @rbharath, May I ask to review once again and guide if is there something else good to add?

Reply:
Ah that makes sense. Should `OpenVINOModel` be a private class then? If we don't ever expect users to interact with it that might make sense

Reply:
Good point! Moved class to `deepchem/utils`